Gemfile

```diff
@@ -30,7 +30,7 @@ gem "puppet-module-win-dev-r#{minor_version}",          require: false, platform
 # https://github.com/puppetlabs/pdk-templates/issues/211
 gem "rake"
 gem "beaker",                                           require: false
-gem "beaker-docker",                                    require: false
+gem "beaker-docker",                                    require: false, git: 'https://github.com/puppetlabs/beaker-docker.git', branch: 'master'
```

Comment:
We should not pull from master. If you need an unreleased feature, we should pull from that specific git sha and have a comment here explaining why and what is the current release so that at the next release this can be removed.